test/integration/state-engine.spec.ts

```diff
@@ -249,13 +249,13 @@ describe('state engine', () => {
 			containerIds,
 		);
 
-		expect(await docker.getNetwork('123_default').inspect())
-			.to.have.property('IPAM')
-			.to.deep.equal({
-				Config: [{ Gateway: '192.168.91.1', Subnet: '192.168.91.0/24' }],
-				Driver: 'default',
-				Options: {},
-			});
+		const customNet = await docker.getNetwork('123_default').inspect();
+		expect(customNet).to.have.property('IPAM');
+		expect(customNet.IPAM).to.have.property('Config');
+		expect(customNet.IPAM.Config).to.have.lengthOf(1);
```

Comment:
Fwiw you can combine these like
```suggestion
		expect(customNet.IPAM).to.have.property('Config').that.has.lengthOf(1);
```
and I think it gives slightly better error messages on failures